src/main/java/org/mockito/internal/session/DefaultMockitoSessionBuilder.java

```diff
@@ -0,0 +1,33 @@
+package org.mockito.internal.session;
+
+import org.mockito.MockitoSession;
+import org.mockito.internal.framework.DefaultMockitoSession;
+import org.mockito.internal.util.ConsoleMockitoLogger;
+import org.mockito.quality.Strictness;
+import org.mockito.session.MockitoSessionBuilder;
+
+public class DefaultMockitoSessionBuilder implements MockitoSessionBuilder {
+
+    private Object testClassInstance;
+    private Strictness strictness;
+
+    @Override
+    public MockitoSessionBuilder initMocks(Object testClassInstance) {
```

Comment:
Yes technically it is indeed fields ;)
But `initMocks` is still inacurate. Ideas : `initAnnotated` (thanks to @raphw), `processAnnotations`, `initFields`. What I'd like is to give these users the hint that it's not only about mocks only, but about mockito annotations in general. And then in the javadoc one can mention the `AnnotationEngine`.